compiler/build.gradle

```diff
@@ -37,8 +37,9 @@ dependencies {
     implementation(project(":ownerContext"))
     implementation(project(":translator"))
     implementation(project(":translator-string"))
-    implementation(project(":idl-declarations"))
-    implementation(project(":idl-models"))
+    implementation(project(":idl:idl-declarations"))
```

Comment:
it's better when project still has unique flat uid so that we can change location of some dependency without changing all dependents throughout whole project.

you can take a look how it's done in kotlin - it's in settings.gradle